docs/articles/expensify-classic/integrations/travel-integrations/Lyft.md

```diff
@@ -1,5 +1,33 @@
 ---
-title: Coming Soon
-description: Coming Soon
+Lyft & Expensify
+Setting up and using Expensify's Lyft integration
 ---
-## Resource Coming Soon!
+<!-- The lines above are required by Jekyll to process the .md file -->
+
+# Overview
+<!--
+You can link Expensify directly to your Lyft account. This means that your receipts for work-related Lyft rides will populate automatically in Expensify. 
+
+-->
+
+# How to connect a Lyft Business profile for individual use
+<!--
+If you use Lyft for work, you can connect your business profile to Expensify to have your Business-related trips populate directly in your Expensify account.
+1. Open the Lyft mobile app and tap your profile photo in the upper left. 
+2. Go to Settings.
+3. Select Business profile, and create one if you don’t already have one. 
+4. Tap Expense management and choose Expensify. You'll be asked to enter your email, make sure it matches the email you use to access Expensify.
+You're all set! Switch into business profile mode each time you take a ride for work, and your receipts will automatically populate in Expensify. 
+
+-->
+
+# How to set up Lyft for Business
+<!--
+If you have a Lyft for Business account for your company, setting up the Expensify and Lyft integration for your whole team is very straightforward:
+1. Open your Lyft for Business portal from a web browser.
+2. Navigate to People > Business Profile (+) and input each team member's work email.
+3. Once invited, each employee will receive an email prompting Lyft Business Profile setup. 
+4. After they set up their profiles, employees can then switch into business profile mode each time they take a ride for work, and their receipts will automatically populate in Expensify.
+Now, when any employee completes a ride on a Lyft Business Profile, it will show up in the "Rides" section of your Lyft for Business portal.
+
+-->
```

Comment:
```suggestion
If you have a Lyft for Business account for your company, setting up the Expensify and Lyft integration for your whole team is very straightforward:
1. Open your Lyft for Business portal from a web browser.
2. Navigate to People > Business Profile (+) and input each team member's work email.
3. Once invited, each employee will receive an email prompting Lyft Business Profile setup. 
4. After they set up their profiles, employees can then switch into business profile mode each time they take a ride for work, and their receipts will automatically populate in Expensify.
Now, when any employee completes a ride on a Lyft Business Profile, it will show up in the "Rides" section of your Lyft for Business portal.
```